tests/dataimportcron_test.go

```diff
@@ -455,6 +456,27 @@ var _ = Describe("DataImportCron", Serial, func() {
 		waitForDigest()
 	})
 
+	It("Should add an appropriate node selector to poller when specfiying architecture with pullMethod node", func() {
+		By("Create DataImportCron with only initial poller job")
+		cron = utils.NewDataImportCron(cronName, "5Gi", scheduleOnceAYear, dataSourceName, importsToKeep, *reg)
+		retentionPolicy := cdiv1.DataImportCronRetainNone
+		cron.Spec.RetentionPolicy = &retentionPolicy
+		cron.Spec.Template.Spec.Source.Registry.Platform = &cdiv1.PlatformOptions{Architecture: "test"}
+
+		cron, err := f.CdiClient.CdiV1beta1().DataImportCrons(ns).Create(context.TODO(), cron, metav1.CreateOptions{})
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Ensuring that nodeSelector was added")
+		pollerPodName := naming.GetResourceName("poller-"+cron.Name, string(cron.UID)[:8])
```

Comment:
Not a big deal but you don't have to use naming.GetResourceName in tests, you know upfront what the name is going to be